prow/cmd/branchprotector/protect_test.go

```diff
@@ -42,22 +42,31 @@ func TestOptions_Validate(t *testing.T) {
 			name: "all ok",
 			opt: options{
 				config: "dummy",
-				github: flagutil.GitHubOptions{TokenPath: "fake"},
+				github: flagutil.GitHubOptions{
+					GitEndpoint: "https://github.com",
```

Comment:
The fact that we're having to change branchprotector unit tests feels wrong to me...

Reply:
Do we need a resolution for this? I agree